roofit/roofitcore/src/RooAbsReal.cxx

```diff
@@ -1660,14 +1663,20 @@ void RooAbsReal::plotOnCompSelect(RooArgSet* selNodes) const
 /// <tr><td> `Components(const RooArgSet& compSet)` <td> As above, but pass a RooArgSet of the components themselves.
 ///
 /// <tr><th><th> Plotting control
-/// <tr><td> `DrawOption(const char* opt)`     <td> Select ROOT draw option for resulting TGraph object
+/// <tr><td> `DrawOption(const char* opt)`     <td> Select ROOT draw option for resulting TGraph object. Currently supported options are "F" (fill), "L" (line), and "P" (points). <i>Caution: Choosing option "P" will cause RooFit to plot (and treat) this pdf as if it were data! This is intended for plotting "corrected data"-type pdfs such as "data-minus-background" or unfolded datasets.</i> 
```

Comment:
Can you break the line in two / three?
Try doxygen's
```
\note Choosing option "P"
or
\warning Choosing option "P"
```
Doxygen indents, colors, and adds either *Note* or *Warning* to the paragraph.